server/views/en/sharemail.jade

```diff
@@ -1,35 +1,39 @@
-            tr
-              td(style="padding: 0.5em 1.5em; width: 100%;")
-                a(href=url, style="display: block; width: 60%; margin: 0 auto; padding: 1.2em 2em; background-color: #34A6FF; border-radius: 5px; color: #FFF; text-decoration: none; text-align: center; font-size: 1em; font-weight: bold; cursor:pointer;")
-                  | #{localization.t('link ' + type + ' content')}.
-            if type == 'folder'
-              tr
-                td(style="padding: 1.5em; color: #999;")
-                  | You can:
-                  ul(style="margin: 0;")
-                    li Download files from this folder
-                    if type == 'folder' && rule.perm == 'rw'
-                      li Add new files to this folder
-                    li Subscribe to change notifications of this folder
-      tr(style="padding-top: 10px; text-align: center; font-size: 0.85em; font-style: italic; color: #999;")
-        td
-          p
-            | Sent from&nbsp;
-            a(href="http://cozy.io", style="color: #34A6FF; text-decoration: none;") #{displayName}'s Cozy
-            | .
+              td
+                table(cellpadding="0" cellspacing="0" border="0", style="width: 100%; max-width: 600px; background-color: #FFF; border-radius: 8px; border: 1px solid #DDD;")
+                  tr
+                    th(style="border-radius: 8px 8px 0 0; border-bottom: 1px solid #DDD;")
+                      img(src="http://cozy.io/assets/images/cozy-logo-gradient.svg" width="63" height="48" style="padding: 12px 0; margin: auto;")
+                  tr
+                    // TODO: add user's email inside the brackets #{displayEmail}
+                    td(style="padding: 1.5em;") #{displayName} (#{displayEmail}) shared a #{localization.t(type)} called "#{doc.name}" with you via Cozy Cloud.
+                  tr
+                    td(style="padding: 0.5em 1.5em 2em; width: 100%;")
+                      a(href=url, style="display: block; width: 60%; margin: 0 auto; padding: 1.2em 2em; background-color: #34A6FF; border-radius: 5px; color: #FFF; text-decoration: none; text-align: center; font-size: 1em; font-weight: bold; cursor:pointer;")
+                        | View #{localization.t(type}
+                  if type == 'folder'
+                    tr
+                      td(style="padding: 0em 1.5em 1.5em; color: #999;")
+                        | You can:
+                        ul(style="margin: 0;")
+                          li Download files from this folder
+                          if type == 'folder' && rule.perm == 'rw'
+                            li Add new files to this folder
+                          li Subscribe to change notifications of this folder
+            tr(style="padding-top: 10px; text-align: center; font-size: 0.85em; font-style: italic; color: #999;")
+              td
+                p
+                  | Sent from
```

Comment:
Could you add a " " (space) after the "from" at the end of the line ? Instead it will result as *Sent fromuser's Cozy.* :smile:.